integration/spark2/src/main/scala/org/apache/spark/sql/CarbonEnv.scala

```diff
@@ -237,6 +237,21 @@ object CarbonEnv {
     getCarbonTable(tableIdentifier.database, tableIdentifier.table)(sparkSession)
   }
 
+  /**
+   * This method returns corresponding CarbonTable, it will return None if it's not a CarbonTable
+   */
+  def getCarbonTableOption(
```

Comment:
There is already a `getCarbonTable` func defined in `CarbonEnv`, is this needed??

Reply:
the getCarbonTable will throw a exception when get a non-carbon table